frontend/src/meeting-detail/RegistrationForm.jsx

```diff
@@ -43,7 +43,7 @@ export default function RegistrationForm({ isDisabled = false }) {
     })
     .catch((error) => {
       console.log(error);
-      setError(`An error has occurred: ${error.message}`);
+      // setError(`An error has occurred: ${error.message}`);
```

Comment:
I believe that this line can be kept (don't comment out).
Because it can catch some errors from the network or the server, errors don't just come from the form (user input).

Reply:
No Probelm！